### PR TITLE
doc firewall cfg

### PR DESCRIPTION
**Pull request by clnperez**
Add a section about which ports need to be opened up if you're running
the automation from behind a firewall.

Signed-off-by: Christy Norman <christy@linux.vnet.ibm.com>

### CODE REVIEW

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/132#" title="Author self-approved">clnperez</a>*
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **bpradipt** after the PR has been reviewed.
You can assign the PR to them by writing `/assign @bpradipt` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by clnperez** [comment]
not sure if there are others needed. @yussufsh or @Prajyot-Parab can you comment?
cc @bpradipt

**Comment by clnperez** [comment]
Moved the update from the quickstart doc to the automation host doc.

File: docs/automation_host_prereqs.md
Comment (bpradipt) [inline docs/automation_host_prereqs.md:20]: Webconsole is only 443 afaik. 6443 is only for CLI. Can you please re-confirm ..

File: docs/automation_host_prereqs.md
Comment (yussufsh) [inline docs/automation_host_prereqs.md:19]: Don't think web console need 6443. It is only for the CLI.

**Comment by clnperez** [comment]
Updated as per suggestions, thanks.

**Comment by bpradipt** [comment]
@clnperez can you please rebase. There are no further updates needed.

**Comment by clnperez** [comment]
@bpradipt rebased

**Review by yussufsh — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/132#" title="Author self-approved">clnperez</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/132#pullrequestreview-523036620" title="LGTM">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by bpradipt — APPROVED** [review]
/lgtm

**Comment by ltccci** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/132#pullrequestreview-523047348" title="LGTM">bpradipt</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/132#" title="Author self-approved">clnperez</a>*, *<a href="https://github.com/ocp-power-automation/ocp4-upi-powervs/pull/132#pullrequestreview-523036620" title="LGTM">yussufsh</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=ocp-power-automation%2Focp4-upi-powervs).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/ocp-power-automation/ocp4-upi-powervs/blob/master/OWNERS)~~ [bpradipt,yussufsh]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by bpradipt** [comment]
This will need backport to release-4.5 and release-4.6 . @clnperez will you be ok to do the backports ?

**Comment by clnperez** [comment]
@bpradipt sorry -- missed that. do you still need backports?

**Pull request merged**
